File: official/docs/java/current/addresses/create-and-verify.java
package addresses;

import com.easypost.exception.EasyPostException;
import com.easypost.model.Address;
import com.easypost.service.EasyPostClient;

import java.util.HashMap;

public class CreateAndVerify {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient(System.getenv("EASYPOST_API_KEY"));

        HashMap<String, Object> addressParams = new HashMap<String, Object>();

        addressParams.put("street1", "417 Montgomery Street");
        addressParams.put("street2", "FL 5");
        addressParams.put("city", "San Francisco");
        addressParams.put("state", "CA");
        addressParams.put("zip", "94104");
        addressParams.put("country", "US");
        addressParams.put("company", "EasyPost");
        addressParams.put("phone", "415-123-4567");

        Address address = client.address.createAndVerify(addressParams);

        System.out.println(address);
    }
}

File: official/docs/node/current/addresses/create-and-verify.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const address = await client.Address.createAndVerify({
    street1: '000 unknown street',
    city: 'Not A City',
    state: 'ZZ',
    zip: '00001',
    country: 'US',
    email: 'test@example.com',
    phone: '5555555555',
  });

  console.log(address);
})();
